Formula/l/lazyssh.rb

```diff
@@ -0,0 +1,24 @@
+class Lazyssh < Formula
+  desc "Terminal-based SSH manager"
+  homepage "https://github.com/Adembc/lazyssh"
+  url "https://github.com/Adembc/lazyssh/archive/refs/tags/v0.2.1.tar.gz"
+  sha256 "ada257fb07db602e92c9c2a3183f267b63fb8e1bf80c4d3292461a003aa794d5"
+  license "Apache-2.0"
+  head "https://github.com/Adembc/lazyssh.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X main.version=#{version}
+      -X main.gitCommit=brew
```

Comment:
```suggestion
      -X main.gitCommit=#{tap.user}
```

Reply:
It expects a short hash, so it only accepts 7 or less characters.

Reply:
got you 👍